cli/data_preparation/waymo/create_data.py

```diff
@@ -62,35 +76,39 @@ def _fill_infos(root_path, frames, split="train", nsweeps=1):
 
         ref_pose = np.reshape(ref_obj["veh_to_global"], [4, 4])
         _, ref_from_global = veh_pos_to_transform(ref_pose)
-
         info = {
             "path": lidar_path,
             "anno_path": ref_path,
             "token": frame_name,
             "timestamp": ref_time,
+            "veh_to_global": ref_pose,
             "sweeps": [],
+            "frame_time": int(ref_obj['frame_name'].split("_")[-1]),
+            "context_name": ref_obj['scene_name']
         }
 
+        sequence_id = int(frame_name.split("_")[1])
+        frame_id = int(frame_name.split("_")[3][:-4])  # remove .pkl
         if split != "test":
             # read boxes
+
             annos = ref_obj["objects"]
             num_points_in_gt = np.array([ann["num_points"] for ann in annos])
             gt_boxes = np.array([ann["box"] for ann in annos]).reshape(-1, 9)
             difficulty = np.array([ann["detection_difficulty_level"] for ann in annos])
-
+            gt_ids = np.array([ann["name"] for ann in annos])
             gt_names = np.array([TYPE_LIST[ann["label"]] for ann in annos])
```

Comment:
Is gt_ids and gt_names the same?

Reply:
gt_ids is unique tracking_id and gt_names id class names